docs/release_notes.adoc

```diff
@@ -39,6 +39,14 @@ subscriber.receive(_) >> _
 - `IterationInfo` now has a property for the `iterationIndex` and one for a map of data variable names to values.
   This is typically not interesting for the average user, but might be helpful for authors of Spock extensions.
 
+- Data driven features are now unrolled by default. `@Unroll` can still be used to specify a custom naming pattern.
+
+- `@Rollup` can now be used on feature and spec level to explicitly roll up any feature where the reporting of single
+  iterations is not wanted.
+
+- The system property `spock.globalRollup` can be set to `true` to roll up all features by default if not overwritten
+  by explicit `@Unroll` annotations and thus reinstate the pre Spock 2.0 behaviour.
```

Comment:
Maybe it would be good to suggest that - in general - after testing phrase "plain" `@Unroll` annotations could be safely removed (here or in `data_driven_testing.adoc`)?

Reply:
What do you mean be "after testing phrase"?
I agree to mention the possibility to remove plain `@Unroll` in release notes, not in the normal docs though imho, I'm just not sure what you meant by that phrase.
I just added a sentence to the "Data driven features are now unrolled by default" bullet point, unless you meant something different.

Reply:
> What do you mean be "after testing phrase"?

I might be more precise (and no to make a type :-) ) - a phase when you testing if Spock 2 works fine in your project.
So, you upgrade Spock to 2.0 in your project. You verify that everything works fine. You remove the annotation (and possibly tests that number of tests didn't decrease).